src/main/resources/application.yml

```diff
@@ -46,6 +46,7 @@ spring:
       enabled: false
 
   jpa:
+    database-platform: org.hibernate.dialect.Oracle10gDialect
```

Comment:
moved to here since we're running h2 in oracle mode so need the oracle dialect instead